kyo-prelude/shared/src/main/scala/kyo/kernel/ArrowEffect.scala

```diff
@@ -0,0 +1,289 @@
+package kyo.kernel
+
+import internal.*
+import kyo.Tag
+import scala.annotation.tailrec
+import scala.util.control.NonFatal
+
+abstract class ArrowEffect[-I[_], +O[_]] extends Effect
```

Comment:
The files are properly renamed in the git history but github doesn't render the rename. The individual commits should be easier to review.

Reply:
I think your rebase may have broke this.

Reply:
sorry, fixed now!